### ISSUE TITLE
Enhancement: Link validation check

### ISSUE THREAD

**Issue by rica-carv** [Jimmi08/links_page#18]
Hi. Is there a way that the plugin can do a link check when the user loads the links page?
So that invalid links are not displayed?

Or maybe a solution that from times to times, the plugin can perform a backend links check, and disable the invalid ones?

**Comment by Jimmi08** [comment]
I suppose that most CMS have functionality report broken link. And then you/admin check it and delete it.

If you know how to do this, you can try. This is old plugin, I don't plan to extended its functionality (mainly because I don't how, I am glad it works with version 2).

I use it because I need it now but in future I would like to use(get inspiration from) Weblinks plugin from Nuke for something like this. But it seems that in far far future now.

Does have core Sitelinks this functionality?

**Comment by rica-carv** [comment]
I've just tested with core sitelinks, and no, if a link is invalid inside sitelinks, it only shows an invalid page. So there's no link check there...

Regarding the links plugin, i thought it was yours, so my question. But ok, i'll plan to refit it, so if you can show me some page where i can peek weblinks from nuke, maybe i'll get new ideias too...

**Comment by Jimmi08** [comment]
No, this is original plugin from version 1. it was as core plugin before...

**Comment by rica-carv** [comment]
@Jimmi08 Sorry for bringing this up, but you've stated that "...in future I would like to use(get inspiration from) Weblinks plugin from Nuke for something like this. But it seems that in far far future now."
If i can ask a question, what would you see from Nuke Weblinks on this plugin?
The top rated links? The most popular links? Or is it something else?
Just to see if we can do something about it....

P.S.: As far i know, PHP-NUKE is not updated since 2013... 😉

**Comment by Jimmi08** [comment]
@rica-carv it's just about way of coding and personal preferencies. Nothing more. LP is a bit complicated...  Yes, that it's old, but it's Czech better version called UnNuke, maybe else older and I still maintain it and I will be for next some years. It's more about my customs not functionality. For me it's easier to maintain 2 similar plugins (with the same logic and maybe with the same tables) then totally different LP plugin. 

By the way, I would like if you take care of this plugin if you want.  You can change author to you etc with next version. No problem.

**Comment by rica-carv** [comment]
@Jimmi08 I could take care of it, but i won't change author, etc., since you've had all the work to bring it to v2, so for me, it's more yours than anything... 😉

**Comment by Jimmi08** [comment]
@rica-carv It's nice, but you can do it. So you should create new directory or something - I don't know what happens when I delete this one. Thank you very much

**Comment by rica-carv** [comment]
@Jimmi08 [I think it's no problem...](https://help.github.com/articles/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility/#deleting-a-public-repository)
